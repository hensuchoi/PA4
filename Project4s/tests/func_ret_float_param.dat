funct: foo
param: float, 5.6
