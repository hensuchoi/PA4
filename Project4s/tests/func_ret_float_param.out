Result: 5.600000e+00
